### ISSUE TITLE
Error running script

### ISSUE THREAD

**Issue by sarah-gao** [mfhamir/apple_mobility_2021#35]
```
Error: 'output/applemobilitytrends-2021-09-26_Alabama.csv' does not exist in current working directory.
```
https://github.com/mfhamir/apple_mobility_2021/blob/e1a4b0705b1a61c46618baeb9b1afd60214dbe01/code/02_countup_cities_source.R#L13

**Comment by mfhamir** [comment]
https://github.com/mfhamir/apple_mobility_2021/commit/27edd3962a095e35ce11094cb93495ad393e6dfd

**Comment by sarah-gao** [comment]
This specific error looks fixed in this latest push, but please see #41 for an error with your 01 script.
